classes/install/form/InstallForm.inc.php

```diff
@@ -79,6 +79,7 @@ function __construct($request) {
 			'informix' => array('ifx', 'Informix'),
 			'sybase' => array('sybase', 'Sybase'),
 			'odbc' => array('odbc', 'ODBC'),
+			'pdo_sqlsrv' => array('pdo_sqlsrv', 'PDO SQL Server')
```

Comment:
If this is simply a PDO connection, and doesn't need to know anything particular about SQL Server, then why not just call it "PDO" here? It's possible that other connections could even be used. (I think this will only be useful if we can get rid of SQL Server-specific SQL formulations but so far I think that's possible.)

Reply:
(i.e. use the same driver name that ADODB expects, rather than `pdo_sqlsrv`)